docs/decisions/0013-custom-groups-for-enterprise-learners.rst

```diff
@@ -0,0 +1,152 @@
+0013 Creating Custom Groups for Enterprise Customers
+##############################
+
+Status
+******
+
+**Pending**
+
+Terminology
+*******
+*Enterprise Group* - Collection of enterprise customer users connected to a policy/subsidy
+
+*Membership* - Relationship between enterprise learner and enterprise group
+
+Context
+*******
+
+Groups is a new feature that will allow admins the flexibility to control and distribute funding. At a high level, a group is a subset of learners within an organization that is tied to an an enterprise customer's policy. A customer can have multiple policies and groups, and a learner can be in multiple groups. These changes will improve the user management and budgeting from the admin perspective, as well as adding personalization to the learner experience. Groups will be associated with a learner credit budget. In the past, we’ve used workarounds to sidestep this functionality, but with this new implementation, we will increase scalability and personalization.
```

Comment:
ditto, its more than just control and distribute funds. I'd add things like analytics segmentation, bulk actions, access control, as well as budgets. They are not tied to the policy as much as policies can make use of groups.